plugins/scenario/utils/hreForBase.ts

```diff
@@ -36,20 +27,7 @@ waffle type extension: https://github.com/nomiclabs/hardhat/blob/master/packages
 */
 declare module 'hardhat/internal/core/runtime-environment' {
   interface Environment {
-    waffle: {
-      provider: MockProvider;
-      deployContract: (
-        signer: Signer,
-        contractJSON: ContractJSON,
-        args?: any[],
-        overrideOptions?: providers.TransactionRequest
-      ) => Promise<Contract>;
-      solidity: typeof solidity;
-      link: typeof link;
-      deployMockContract: (signer: Signer, abi: any[]) => Promise<MockContract>;
-      createFixtureLoader: typeof createFixtureLoader;
-      loadFixture: typeof loadFixture;
-    };
+    waffle: any;
```

Comment:
I suspect we can delete this `waffle` key entirely, since we're no longer attaching waffle matchers to the environment.

Reply:
Maybe informative to leave it though, I'm not going to mess with it